src/promise.js

```diff
@@ -88,6 +88,7 @@ function Promise(executor) {
         // by enclosed function which is added later
         // by build process (tools/build.js!buildBrowser)
         this.currentHadronTest = _global.currentHadronTest;
+        this.creationStack     = new Error().stack;
```

Comment:
This is awesome! But I think it may really slow down test runs. Also, won't work in IE, at least older versions. It seems like this should be toggled by the `longStackTraces` flag.